backends/cuda-gen/ceed-cuda-gen-qfunction.c

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2017-2018, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory. LLNL-CODE-734707.
+// All Rights reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
+#include <ceed-backend.h>
+#include <string.h>
+#include <stdio.h>
+#include "../cuda/ceed-cuda.h"
+#include "ceed-cuda-gen.h"
+
+static int CeedQFunctionApply_Cuda_gen(CeedQFunction qf, CeedInt Q,
+                                       CeedVector *U, CeedVector *V) {
+  int ierr;
+  Ceed ceed;
+  ierr = CeedQFunctionGetCeed(qf, &ceed); CeedChk(ierr);
+  return CeedError(ceed, 1, "Backend does not implement QFunctionApply");
+}
```

Comment:
One suggestion - Do we want to delegate this to a different CUDA backend just in case `CeedQFunctionApply` is important to the user's code? I'm not sure it's possible, but I thought I would ask. The user can always make a second CEED to do it themselves.

Reply:
It's not so simple to delegate because the delegate doesn't use the same qFunction file for the moment, I can add the delegation to #288 .